dashboard/common/db_util.py

```diff
@@ -12,16 +13,29 @@
 
 logger = logging.getLogger(__name__)
 
+def canvas_id_to_prefixed_id(canvas_id):
+    # Add the DATA_WAREHOUSE_ID_PREFIX and to the id
+    canvas_id = str(canvas_id)
+    prefixed_id = str(settings.DATA_WAREHOUSE_ID_PREFIX)[0:-len(canvas_id)] + canvas_id
+    return prefixed_id
+
+def prefixed_id_to_canvas_id(prefixed_id):
+    # Remove the DATA_WAREHOUSE_ID_PREFIX and just return the id
```

Comment:
However, settings. DATA_WAREHOUSE_ID_PREFIX was not used directly in line 24-26?

Reply:
I've rewritten the comments explain what `canvas_id_to_prefixed_id` and `prefixed_id_to_canvas_id` do (hopefully better!)